src/js/containers/award/AwardContainer.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+  * AwardContainer.jsx
+  * Created by Emily Gullo 01/19/2017
+  **/
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import Award from 'components/award/Award';
+
+import * as SearchHelper from 'helpers/searchHelper';
+import * as awardActions from 'redux/actions/award/awardActions';
+
+const propTypes = {
+    setSelectedAward: React.PropTypes.func,
+    params: React.PropTypes.object
+};
+
+class AwardContainer extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.getSelectedAward = this.getSelectedAward.bind(this);
+    }
+
+    getSelectedAward() {
```

Comment:
As discussed in person, this container should be responsible for fetching the award data on mount (`componentDidMount`) and pass the award data down the child components.

Additionally, this container should monitor for changes to the award ID URL param in order to update when the URL changes.